src/main/java/enums/Color.java

```diff
@@ -0,0 +1,32 @@
+package enums;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Description.
```

Comment:
I would think a top level comment could provide all the necessary detail and then the comments around the enum values are somewhat redundant

Reply:
IDE default - forgot it was there.